proton-c/bindings/javascript/CMakeLists.txt

```diff
@@ -192,6 +192,7 @@ target_link_libraries(qpid-proton-bitcode)
 
 
 # Compile the send-async.c and recv-async.c examples into JavaScript
+include_directories(${PN_PATH}/../examples/c/include)
```

Comment:
It might be better to avoid relative paths and use

${Proton_SOURCE_DIR}/examples/c/include

My spidey sense tells me that this would be more robust in the face of changes, but I can't really substantiate that.
This would also apply to the relative paths in the lines below.